VKSample/Modules/GroupsModule/GroupsTableViewController.swift

```diff
@@ -3,5 +3,61 @@
 
 import UIKit
 
-/// Screen with groups list
-final class GroupsTableViewController: UITableViewController {}
+/// Screen with groups list.
+final class GroupsTableViewController: UITableViewController {
+    typealias GroupHandler = (Group) -> ()
+
+    // MARK: - Private Constants.
+
+    private enum Constants {
+        static let groupsCellID = "GroupCell"
+    }
+
+    // MARK: - Public properties.
+
+    var subscribedGroupHandler: GroupHandler?
```

Comment:
```suggestion
   private var subscribedGroupHandler: GroupHandler?
```
раз ты устанавливаешь значение через метод configure, то незачем делать свойство публичным